modules/cnf-sample-single-numa-policy-from-pp.adoc

```diff
@@ -32,5 +32,10 @@ spec:
     realTime: true
 ----
 
-<1> This should match the `MachineConfigPool` that you want to configure the NUMA Resources Operator on. For example, you might have created a `MachineConfigPool` named `worker-cnf` that designates a set of nodes that run telecommunications workloads.
-<2> The `topologyPolicy` must be set to `single-numa-node`. Ensure that this is the case by setting the `topology-manager-policy` argument to `single-numa-node` when running the PPC tool.
\ No newline at end of file
+<1> This should match the `MachineConfigPool` that you want to configure the NUMA Resources Operator on. For example, you might have created a `MachineConfigPool` named `worker-cnf` that designates a set of nodes that run telecommunications workloads. It should match the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR configured later in "Creating the NUMAResourcesOperator custom resource".
+<2> The `topologyPolicy` must be set to `single-numa-node`. Ensure that this is the case by setting the `topology-manager-policy` argument to `single-numa-node` when running the PPC tool.
++
+[NOTE]
+====
+For hosted control plane clusters, the `machineConfigPoolSelector` does not have any functional effect. Node association is instead determined by the specified `nodePool`.
```

Comment:
```suggestion
For hosted control plane clusters, the `machineConfigPoolSelector` does not have any functional effect. Node association is instead determined by the specified `NodePool` object.
```
In the hosted control planes documentation, the spelling is typically `NodePool`.